config/MasterTranslations.xml

```diff
@@ -691,16 +691,16 @@
 		</Translation>
 		<Translation name="CP_vehicle_setting_optionalSowingMachineEnabled_tooltip">
 			<Text language="de"><![CDATA[Schaltet die zusätzliche Saatfunktion des Gerätes ein oder aus.]]></Text>
-			<Text language="en"><![CDATA[Turns on or off the additional sowing function of the machine]]></Text>
+			<Text language="en"><![CDATA[Turns on or off the additional sowing function of the machine.]]></Text>
 		</Translation>
 	</Category>
 	<Category name="Vehicle settings combine">
 		<Translation name="CP_vehicle_setting_automaticCutterAttach_title">
-			<Text language="de"><![CDATA[Automatisches Schneidwerk ankuppeln]]></Text>
+			<Text language="de"><![CDATA[Automatisches Schneidwerkankuppeln]]></Text>
```

Comment:
Eventuell sollte man das ändern in "Schneidwerk automatisch ankuppeln" - zusammen geschrieben sieht und hört sich das falsch an.